src/player.py

```diff
@@ -19,11 +20,15 @@
 
 MAX_INVENTORY_CAPACITY = 32
 
+DEFAULT_PLAYER_X = (SCREEN_WIDTH // 2 // TILE_SIZE) * TILE_SIZE + TILE_SIZE / 2
+DEFAULT_PLAYER_Y = (SCREEN_HEIGHT // 2 // TILE_SIZE) * TILE_SIZE + TILE_SIZE / 2
+DEFAULT_PLAYER_POS = (DEFAULT_PLAYER_X, DEFAULT_PLAYER_Y)
```

Comment:
Subjective style nit, feel free to ignore it:

```suggestion
MAX_X_TILES = SCREEN_WIDTH // TILE_SIZE
MAX_Y_TILES = SCREEN_HEIGHT // TILE_SIZE
DEFAULT_PLAYER_POS = (
    (MAX_X_TILES // 2) * TILE_SIZE + TILE_SIZE / 2,
    (MAX_Y_TILES // 2) * TILE_SIZE + TILE_SIZE / 2,
)
```

Also: could the `setting` provide a non-even `TILE_SIZE`? Should we be using `// 2` there for good measure?

Reply:
The Issue was closed surely by mistake, reopened for the time being.

TILE_SIZE should represent the size of the tile in pixels, its a pretty CONST CONST, since all the art work (spritesheets) heavily depend on it and will not change, for clarity's sake though using `//` to ensure remaining on INT land is better.

Finally I decided using a variable for the 16x9 grid, GRID_COLS and GRID_ROWS, so we can avoid multiplying/dividing in different places. I also removed the HALF_TILE to the calculation, since the game, despite having a grid, is not grid bound. Now Capybaba starts in the dead center (in the junction of 4 tiles).

Also added a grid transparent outline when inside game for clarity.

I believe most all nits are worth it and therefore commited.